src/main/webapp/i18n/de/lecture.json

```
@@ -100,7 +100,16 @@
             "pdfPreview": {
                 "title": "Anhang",
                 "attachmentIDError": "Ungültiger Anhang oder ungültige Anhangs-ID.",
-                "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID."
+                "attachmentUnitIDError": "Ungültige Dateieinheit oder ungültige Dateieinheits-ID.",
+                "removePageButton": "Entferne Seiten",
```

Could we change this to "Seiten entfernen"? This would be more consistent with the rest of our translations, especially the button next to it :)

```suggestion
                "removePageButton": "Seiten entfernen",
```

---

Sure :)